tools/generate-sponsors.ts

```diff
@@ -184,14 +185,23 @@ async function main(): Promise<void> {
     __dirname,
     '../packages/website/data/sponsors.json',
   );
-  fs.writeFileSync(
-    rcPath,
-    JSON.stringify(
-      allSponsorsConfig,
-      (_, value: unknown) => value ?? undefined,
-      2,
-    ) + '\n',
+  fs.writeFileSync(rcPath, await stringifyObject(allSponsorsConfig));
+}
+
+async function stringifyObject(data: unknown): Promise<string> {
+  const config = await prettier.resolveConfig(
+    path.join(__dirname, '../.prettierrc.json'),
```

Comment:
nitpick
```suggestion
    path.resolve(__dirname, '..', '.prettierrc.json'),
```

Reply:
you are reviewing not latest version i changed this code